packages/components/src/PassengerCard/PassengerCard.js

```diff
@@ -0,0 +1,112 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BagInformation from './BagInformation';
+import PassengerCardDetail from './PassengerCardDetail';
+import Separator from '../separator/Separator';
+
+type Bag = {|
+  +count: number,
+  +type: string,
+|};
+
+type Props = {|
+  +name: string,
+  +gender: 'female' | 'male' | 'other',
+  +nationality: string,
+  +dateOfBirth: string,
+  +id: string,
+  +insurance: string,
+  +bags: Array<Bag>,
+  +passengerCount: number,
+|};
+
+export default function PassengerCard(props: Props) {
+  const newPassenger = `${props.passengerCount}. Passenger`;
+  const title =
+    props.gender === 'male' ? 'Mr.' : props.gender === 'female' ? 'Ms.' : '';
+  const passengerWithTitle = `${title} ${props.name}`;
+
+  const passenger = props.name ? passengerWithTitle : newPassenger;
+  const { nationality, dateOfBirth, id, insurance, bags } = props;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.containerName}>
+        <Icon name="passenger" />
+        <Text size="large">{passenger}</Text>
+      </View>
+      <View style={styles.containerTop}>
+        <PassengerCardDetail field={nationality} name="Nationality" />
```

Comment:
I would use `label` instead of `name` and `value` instead of `field`. It's little bit confusing

Reply:
Much better now, thanks!